packages/components-react/src/index.test.ts

```diff
@@ -0,0 +1,7 @@
+import { describe, expect, it } from '@jest/globals';
+
+describe('index', () => {
+  it('has a placeholder test', () => {
```

Comment:
We kunnen werken met test.todo placeholders. Op die manier laat de console output van de unit test fase zien waar de placeholders/todo's staan en verdwijnen ze niet uit het zicht.
https://jestjs.io/blog/2019/01/25/jest-24-refreshing-polished-typescript-friendly#testtodo

Reply:
Mooie vondst! Ik heb het aangepast